### PR TITLE
WebSockets Backend

### PR DESCRIPTION
**Pull request by SullyBirashk**
## Features
1. Installed 'Redis' Gem for WebSockets
2. Create Chat Channel 
3. Create Message Model
6. Add /cable route for WebSockets
7. Add Messages Table to database
8. Refactor Facade Test Name
9. Refactor Poro Test
10. Create routes to a Social Page
11. Create Social Controller with Post and Create Request
12. Add Messages Seeded Data 
13. Update User relationship with messages
14. Remove Messages Controller

## Concerns
1. There is no way for us to test this until both Frontend and Backend Applications are running. So we don't know if something needs to be refactored or not! 

## Pre-Submittal Checklist
- [ ] All tests pass on local
- [ ] Visually confirm view functionality
- [x] No prys or save_and_open_page commands remain
- [x] Unnecessary commented code removed

## As a GIF, how does this PR make you feel?
nail biter

### CODE REVIEW

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class MessagesController < ApplicationController
+
+ def create
+   user = User.find_by(params[:name])
```
Comment (megstang) [inline app/controllers/messages_controller.rb:4]: Are these names that are passed in going to be unique?

Comment (megstang) [reply app/controllers/messages_controller.rb:4]: If not, maybe we consider using an id of the user instead of a name?

**Review by megstang — COMMENTED** [review]
@SullyBirashk @samlsmith424 @gregoryjflaherty Seems like y'all might be referencing a blog post when getting this started, which is totally fine. I think the basic setup to get started is there. I just made one comment regarding the uniqueness of the parameter that comes in for looking up your user. At this time, not too much to review, as I know you still need to get some moving pieces going with FE before you can refactor the outline you currently have.

File: Gemfile [hunk 2]
```diff
@@ -11,6 +11,7 @@ gem 'jsonapi-serializer'
 gem 'faraday'
 gem 'rack-cors'
 gem 'active_model_serializers'
+gem 'redis'
```
Comment (samlsmith424) [inline Gemfile:14]: Nice job incorporating this gem!

File: config/routes.rb [hunk 3]
```diff
@@ -8,4 +8,6 @@
       resources :user, except: [:delete, :index]
     end
   end
+
+  mount ActionCable.server => '/cable'
```
Comment (samlsmith424) [inline config/routes.rb:13]: Awesome start for the implementation of websockets.

**Pull request merged**